packaging/nuget/Particular.CodeRules.nuspec

```diff
@@ -2,8 +2,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
     <metadata>
         <id>Particular.CodeRules</id>
-        <version>$version$</version>
+        <version>$version$-beta</version>
```

Comment:
StyleCop analyzer is not v1.0.0, the dependency on the pre-release can be removed. I'll remove it